### PR TITLE
Dungeoneer tweaks/more hermes

### PR DESCRIPTION
**Pull request by MickDick1**
## About The Pull Request

Added 1 healthpot to the dungeoneers room and 1 to the royal torture chamber.
Reinforced the nondoor bars in the sewer entrance to the dungeon to match the door.
Removed the town_dungeon key from the townwatch keyring.
Added four fistfulls of crackers to the torture chamber in the dungeon.
The dungeoneer now accurate to his tutorial intro has an appropriate amount of mammons in his account.
Bare minimum soft armor including a coif (hides under the sack hood), gambeson, and shortboots.
Hermes added to the Dungeoneers bedroom + smithy + inn + merchant's.
Moved the scomm in the dungeon torture chamber slightly.
Gave the dungeoneer 2 medicine skill instead of just 1.

## Why It's Good For The Game

The healthpots are for keeping prisoners alive when stuff happens obviously. Why wouldn't this well-paid sadistic bloke that plays with and breaks his toys not be prepared?

The sewer entrance being casually broken into is annoying.

The big one. The controversial. The town watchmen losing their dungeon keys.
The scoop is that the players here keep playing town watchman like dungeoneer without the stigma. I have not once seen them bring a prisoner to the dungeon. All these nimrods do is constantly mess with the prisoners and meddle in the Dungeoneer's affairs. It's absolutely atrocious behavior that needs to be checked.
The town watch barracks key is still the only key to open the holding cell just above the dungeon. I put a few beds in there, they can cope with it. Not even the dungeoneer has access to it either. This is probably the most important change in this PR out of everything else. Also, there are several other roles that have the town dungeon keys so even if there is no dungeoneer they can pick up the mantle.

Basic rations just make sense. It won't last long between potentially 4 prisoners + dungeoneer. It's the bare minimum.

The dungeoneer tutorial specifically states: "You will never know hunger, thirst, or want for anything with the mammons you make"
>Proceeds to spawn with only 5 zennies to their name

Bare minimum protective gear. Barely affects his drip. Maybe we can make it start darker? IDK.
dungeoneer 1

Hermes for filing mail reports to the king/etc instead of yelling into the scomm or running all the way to court every time.
**THIS WILL CHANGE THE CURRENT DESIGNATIONS OF THE REST OF THE HERMES.**
The court hermes moves up from # 5 to # 6 for example and the dungeoneer hermes ended up being # 2 for some reason.... I looked in the vars and there was no designation to be found. I guess the # of the hermes are autogenerated.

Personal taste about the scomm placement at the dungeon torture chamber (with the mantrap). Moved it to the bottom right corner.

This guy is the ROYAL torturer and dungeon master but he can barely stitch a wound together come on.

tl;dr Dungeoneers should be able capable of focusing on their responsibilities in general with this PR rather than needing to screw about obtaining basic stuff and to stop white knights/nimrods from bothering him so much while he works like they do EVERY SINGLE ROUND.

### CODE REVIEW

**Comment by Red-Honk** [comment]
I'd advise giving them the same gam' that the councilors get for the drip factor.

**Comment by darkincarnate** [comment]
Give him heavy leather gloves on spawn to match the drip

**Comment by MickDick1** [comment]
I already added one hermes so the numbers are all going to change.

So- screw it. I went ahead and tacked on more mail machines to the map so we don't have to go over it again.

The merchant's, smithy, inn (public), and dungeoneer now get a hermes. I'll run around and note the numbers that get assigned.

**Comment by MickDick1** [comment]
bandit 1
dungeoneer 2
village elder 3
boggers 4
steward 5
smithy 6
court 7
merchant 8
magician 9
inn 10
church 11
priest 12

Also, final dungeoneer drip:
dungeoneer 2

**Comment by MickDick1** [comment]
Compromise. It's not in earshot of the keep, it's further away from the town road, it keeps the watchmen out, etc.

dungeon compromise
